config.go

```diff
@@ -9,8 +9,13 @@ import (
 
 // Config for gometalinter. This can be loaded from a JSON file with --config.
 type Config struct { // nolint: aligncheck
-	// A map of linter name to "<command>:<pattern>".
-	Linters map[string]string
+	// A map from linter name -> <LinterConfig|string>.
+	//
+	// For backwards compatibility, the value stored in the JSON blob can also
+	// be a string of the form "<command>:<pattern>", <command> should always
+	// include {path} as the target directory to execute. Globs in <command>
```

Comment:
Looks like this merge conflict wasn't resolved correctly. The part of about needing to include a `{path}` should be removed, `{path}` isn't used anymore